doc/source/ray-observability/ray-distributed-debugger.rst

```diff
@@ -199,7 +199,7 @@ When the app throws an exception:
 - The paused task is listed in the Ray Debugger extension.
 - Click the play icon next to the name of the paused task to attach the debugger and start debugging.
 
-.. image:: ./images/post-moretem.gif
+.. image:: ./images/post-mortem.gif
```

Comment:
Need to confirm this is the correct name of the linked GIF.

Reply:
Has renamed. You could check

<img width="2517" height="393" alt="image" src="https://github.com/user-attachments/assets/1ed00cbd-e36e-4f1c-9fb9-362354646e19" />